sys/include/usb.h

```diff
@@ -24,17 +24,29 @@
 extern "C" {
 #endif
 
+#include "board.h"
+
 /**
  * @defgroup usb_conf USB peripheral compile time configurations
  * @ingroup config
  * @{
  */
 
+/* These can be overridden by boards that should come up with their board
+ * supplier VID/PID pair. Boards should only override this if the RIOT built-in
+ * peripherals are compatible with whatever is usually shipped with that pair
+ * */
```

Comment:
Nit-picking here but does the variable name make sense? If its overridne then its actually not `RIOT`s PID but the board or manufacturer right? Would it make more sense to just have `INTERNAL_PERIPHERAL_PID` (If my understanding of the variable meaning is correct)